AK/CMakeLists.txt

```diff
@@ -59,7 +59,7 @@ swizzle_target_properties_for_swift(FastFloat::fast_float)
 target_link_libraries(AK PRIVATE FastFloat::fast_float)
 
 # FIXME: Make this generic for all imported shared library dependencies and apply globally
-if (BUILD_SHARED_LIBS AND NOT CMAKE_SKIP_INSTALL_RULES AND NOT "${VCPKG_INSTALLED_DIR}" STREQUAL "")
```

Comment:
this could probably be avoided as a patch if we always did a static build (-DBUILD_SHARED_LIBS=OFF) for the tools. given that the only tools builds we have rn are in BuildFuzzers.sh and BuildLagom.sh for the Android build, not really going to sweat it though.